assets/prototype/domain/eventEditor/ticketsList/ticketPriceCalculator/hooks/useOnSubmitPrices.js

```diff
@@ -31,11 +31,20 @@ const useOnSubmitPrices = (existingPrices) => {
 				return;
 			}
 			const { id, ...input } = pick(PRICE_INPUT_FIELDS, price);
+
+			const normalizedPriceFields = {
+				...input,
+				amount: parseFloat(price.amount || 0),
+				isDefault: !! price.isDefault,
+				order: parseInt(price.order, 10),
+				overrides: parseInt(price.overrides, 10),
+				parent: parseInt(price.parent, 10),
```

Comment:
Many of these fields may not be present in `input`. Also, `parent` is `id` (GUID) of another price.

Reply:
oh right... forgot those are guids client side